publicodes/core/test/mécanismes/résoudre-le-cycle.yaml

```diff
@@ -0,0 +1,67 @@
+fx: 
+x:
+  résoudre le cycle: oui
```

Comment:
Super intéressant, ça ressemble à ce qui existe dans Excel
https://www.ablebits.com/office-addins-blog/2015/12/15/circular-reference-excel/
https://support.microsoft.com/fr-fr/office/supprimer-ou-autoriser-une-r%C3%A9f%C3%A9rence-circulaire-8540bd0f-6e97-4483-bcf7-1b49cd50d123

Peut-être que l'on peut s'en inspirer pour le nom de la fonction : « résoudre les références circulaires » ou « activer le calcul itératif »

Ça permet également de différencier l'inversion d'une formule (net → brut) de la résolution de références circulaires (même si c'est le même calcul itératif dans les deux cas ?)

Reply:
Oui, tout à fait ! Merci pour les liens Excel, c'est super intéressant. Je suis tout à fait disposer à changer le nom pour être plus proche de la version Excel :)